backend/development.ini.sample

```diff
@@ -72,6 +72,9 @@ basic_setup.session_secret = change_this_value_please!
 # max upload content length accepted by tracim for file in bytes
 # by default 0 meaning no limit. 104857600 (100MB) is a good value
 ; limitation.content_length_file_size = 104857600
+# max size of all files of a workspace in bytes
+# by default 0 meaning no limit. 1073741824(1GB) is a good value
+; limitation.workspace_size = 1073741824
```

Comment:
`limitation.content_length_file_size` and `limitation.workspace_size` example values in `development.ini.sample` are not the `config.py` applied default value. These values must be the same.

Reply:
I'm not sure of this. default value is "0" meaning not limit, but it make sense to show meaningful not 0 value, like in the comment. 
You think we should just change : 
`; limitation.workspace_size = 1073741824`
to 
`; limitation.workspace_size = 0` ?

@PhilippeAccorsi what do think about this ?

Reply:
> @PhilippeAccorsi what do think about this ?

I don't know. I understand that if I uncomment ths two line, I think its working like at present. But If I want to make limitation, for me its more simple if when I uncomment line, limitation is activated.

Reply:
after discussion with @lebouquetin, decision was made to not change actual code.

Reply:
Ok